homeassistant/helpers/travel_time.py

```diff
@@ -0,0 +1,9 @@
+"""Helper methods for travel_time integrations."""
+from homeassistant.const import ATTR_LATITUDE, ATTR_LONGITUDE
+from homeassistant.core import State
+
+
+def get_location_from_attributes(entity: State) -> str:
```

Comment:
I feel like this is too implementation specific to generalize. Sure, different integrations use the same code but I don't think that it's worth extracting.